proxy/src/serverless/backend.rs

```diff
@@ -571,6 +571,13 @@ impl ConnectMechanism for TokioMechanism {
             "compute_id",
             tracing::field::display(&node_info.aux.compute_id),
         );
+
+        if let Some(query_id) = ctx.get_testodrome_id() {
+            tracing::Span::current()
+                .record("latency=", tracing::field::display(ctx.get_proxy_latency()));
+            tracing::Span::current().record("query_id=", query_id);
```

Comment:
```suggestion
            tracing::Span::current()
                .record("latency", tracing::field::display(ctx.get_proxy_latency()));
            tracing::Span::current().record("query_id", query_id);
```
as well as in the other places

Reply:
Additionally, `record()` does nothing unless the field name has been pre-registered.

https://docs.rs/tracing/latest/tracing/struct.Span.html#method.record